cmd/buildah/inspect.go

```diff
@@ -96,7 +97,12 @@ func inspectCmd(c *cli.Context) error {
 	}
 
 	if c.IsSet("format") {
-		return t.Execute(os.Stdout, buildah.GetBuildInfo(builder))
+		err := t.Execute(os.Stdout, buildah.GetBuildInfo(builder))
+		if err != nil {
```

Comment:
But before I leave, I'll hit you with one of Dan's.  Can this change to:
```
if err := t.Execute(os.Stdout, buildah.GetBuildInfo(builder)); err !=nil {
    return err
}
```

Reply:
fixed.